### ISSUE TITLE
AttributeError: 'NoteFrame' object has no attribute 'nore'

### ISSUE THREAD

**Issue by littlewhitecloud** [falk-werner/note.py#20]
First, this project is really great ngl
I checked it out and I wrote "### Hello" to test, when I took focus on "View" tab, it crashed...
![image](https://user-images.githubusercontent.com/71159641/211486502-346e49d4-ebe7-4fcc-aff7-b4ba99597b70.png)
![image](https://user-images.githubusercontent.com/71159641/211486571-e9f92ca4-5cf7-4881-938b-6f6d9ec07bbd.png)
![image](https://user-images.githubusercontent.com/71159641/211486698-7979652e-5cae-46a3-b208-21f70e37eb07.png)
```
Exception in Tkinter callback
Traceback (most recent call last):
  File "C:\Program Files\Python311\Lib\tkinter\__init__.py", line 1948, in __call__
    return self.func(*args)
           ^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Downloads\note.py", line 374, in <lambda>
    self.text.bind('<KeyRelease>', lambda e: self.update_view())
                                             ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Downloads\note.py", line 387, in update_view
    self.frame.add_css(self.nore.css())
                       ^^^^^^^^^
AttributeError: 'NoteFrame' object has no attribute 'nore'
```

**Comment by falk-werner** [comment]
Thank you for the bug report. We will provide a bugfix release soon (see [PR #21](https://github.com/falk-werner/note.py/pull/21)).

**Comment by falk-werner** [comment]
Bugfix is available via new release [v1.0.1](https://github.com/falk-werner/note.py/releases/tag/v1.0.1).
